jekyll/_docs/enterprise/aws.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: enterprise
+title: "AWS"
+category: [enterprise]
+order: 3
+description: "How to install CircleCI Enterprise on Amazon Web Services (AWS)."
+---
+
+The following step-by-step instructions will guide you through the process of installing CircleCI Enterprise on AWS. If you have any questions as you go through these steps, please contact <enterprise-support@circleci.com>.
+
+<ol>
+  <li>Launch resources on AWS using Terraform (see [here]({{site.baseurl}}/enterprise/aws-manual/) for advanced instructions if you prefer to set up resources manually or using different tools):
```

Comment:
Stuff like this doesn't render correctly, working on a fix to address these issues